api/token.go

```diff
@@ -146,10 +161,46 @@ func (a *API) issueRefreshToken(ctx context.Context, user *models.User) (*Access
 	}, nil
 }
 
+func (a *API) setCookieToken(ctx context.Context, user *models.User, session bool, w http.ResponseWriter) error {
```

Comment:
if you don't explicitly really need the context, I'd consider asking for the `config` in the signature.